tests/Jackalope/Test/Fixture/DBUnitFixtureXML.php

```diff
@@ -414,4 +435,77 @@ protected function ensureTableExists($tableName, $columns)
         return $this;
     }
 
+    private function addVersioningProperties(\DOMDocument $dom, \DOMElement $node, $workspaceName, $id)
```

Comment:
@lsmith77 if you have a better idea how we can do this, i would be glad. it is what we came up with at the hackday and while it works, its a bit cumbersome.